LayoutTests/imported/w3c/web-platform-tests/cookie-store/cookieStore_get_arguments.https.any.serviceworker-expected.txt

```diff
@@ -1,11 +1,11 @@
 
-FAIL cookieStore.get with no arguments returns TypeError assert_unreached: Should have rejected: undefined Reached unreachable code
-FAIL cookieStore.get with empty options returns TypeError assert_unreached: Should have rejected: undefined Reached unreachable code
+PASS cookieStore.get with no arguments returns TypeError
+PASS cookieStore.get with empty options returns TypeError
 PASS cookieStore.get with positional name
 PASS cookieStore.get with name in options
 PASS cookieStore.get with name in both positional arguments and options
 PASS cookieStore.get with absolute url in options
 PASS cookieStore.get with relative url in options
 FAIL cookieStore.get with invalid url path in options assert_unreached: Should have rejected: undefined Reached unreachable code
```

Comment:
It's odd that that this particular sub-test passes for the Window case and fails for the Service Worker case. This is why:

We are doing two checks on the URL passed in. In the Window case, the test passes because the first check detects that this is a Window context and that the complete URL and this URL do not match and correctly returns a TypeError. In the Service Worker case, because this is not a Window context, we do not check if the complete URL and the passed in URL match. And the origins are indeed the same. So our checks don't immediately return a TypeError. 

The spec explicitly says to do the check this way ("If this's relevant global object is a Window object and parsed does not equal url, then return a promise rejected with a TypeError" in https://wicg.github.io/cookie-store/#CookieStore-get). So the test should conform to the spec and not expect an immediate TypeError here in the Service Worker case. I'm looking into changing the test and creating a pull request in the WPT repository.